python/tank/bootstrap/resolver.py

```diff
@@ -671,40 +711,22 @@ def resolve_shotgun_configuration(
             # 1 windows/linux/mac path
             # 2 descriptor
             # 3 sg_descriptor
-
-            path = ShotgunPath.from_shotgun_dict(pipeline_config)
-
-            if path and not path.current_os:
+            if pipeline_config["config_descriptor"] is None:
```

Comment:
All of that logging has been moved to `_create_descriptor` and now having a descriptor means the pipeline configuration is valid.